src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -167,6 +168,8 @@ public final class Helpers
     private static final int PRIME_X = 501125321;
     private static final int PRIME_Y = 1136930381;
 
+    private static final boolean JEI = ModList.get().isLoaded("jei");
```

Comment:
```suggestion
    private static final boolean JEI = !BOOTSTRAP_ENVIRONMENT && ModList.get().isLoaded("jei");
```
This should fix the unit tests breaking I believe, since it's not a classload error, just failing to call `isLoaded()` on ` null`